app/navigation/Navigator.scala

```diff
@@ -88,12 +90,12 @@ class Navigator @Inject()() {
     }
   }
 
-  private def trustHaveAUTRRoute(answers: UserAnswers) = {
+  private def trustHaveAUTRRoute(answers: UserAnswers, af: AffinityGroup) = {
     val condition = (answers.get(TrustRegisteredOnlinePage), answers.get(TrustHaveAUTRPage))
 
     condition match {
       case (Some(false), Some(true)) => routes.WhatIsTheUTRController.onPageLoad(NormalMode)
-      case (Some(false), Some(false)) => routes.TrustNameController.onPageLoad(NormalMode)
+      case (Some(false), Some(false)) => if(af == AffinityGroup.Organisation){ routes.AgentInternalReferenceController.onPageLoad(NormalMode)} else {routes.AgentInternalReferenceController.onPageLoad(NormalMode)}
```

Comment:
Your conditional expression goes to the same page here? Surely one of them should be different?